addons/website/views/snippets/s_facebook_page.xml

```diff
@@ -3,8 +3,12 @@
 
 <!-- Snippet template -->
 <template id="s_facebook_page" name="Facebook">
-    <div class="o_facebook_page">
-        <iframe class="mw-100 o_facebook_page_preview" src="https://www.facebook.com/plugins/page.php?height=70&amp;hide_cover=true&amp;href=https%3A%2F%2Fwww.facebook.com%2FOdoo&amp;show_facepile=false&amp;small_header=true&amp;tabs=&amp;width=500" style="width: 500px; height: 70px; border: medium none; overflow: hidden;" aria-label="Facebook"/>
+    <div class="o_facebook_page o_not_editable">
```

Comment:
What about already dropped snippets?

Reply:
Sure we can add `data-vxml` for version control for this snippet. Thanks for pointing it out.